src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const keysValues = sourceString.split(';');
+
+  const cssObject = {};
+
+  keysValues.forEach((keyValue) => {
+    const trimmedKeyValue = keyValue.trim();
+
+    if (trimmedKeyValue) {
+      const parts = trimmedKeyValue.split(':');
+
+      const property = parts[0].trim();
+
+      const value = parts[1].trim();
```

Comment:
There is a potential issue if 'parts[1]' is undefined due to a malformed style declaration (e.g., 'color:'). This would cause 'trim' to throw an error. It's important to check if 'parts[1]' exists before calling 'trim' on it.